ecs-cli/modules/utils/compose/ecs_params_reader.go

```diff
@@ -98,6 +119,87 @@ func (cd *ContainerDef) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	return nil
 }
 
+// HealthCheck.UnmarshalYAML is a custom unmarshaler for healthcheck that parses
+// both the docker compose and ecs inspired syntaxes
+func (h *HealthCheck) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	// All of the different health check formats are mutually exclusive in each of their fields
+	// This makes parsing simple.
+	healthCheck := HealthCheck{}
+
+	// set default value for retries
+	rawHealthCheck := healthCheckFormat{
+		Retries: 3,
```

Comment:
This is an integer field, so it will default to zero unless I provide a different default. Whereas the other fields are strings, so no default is needed because if they are unset they will be empty strings.

Reply:
Good idea 👍